operations/configuration.md

```diff
@@ -316,64 +316,124 @@ Returns all tax environments supported by the API.
 ```javascript
 {
     "TaxEnvironments": [
-        {
-            "Code": "AU",
-            "CountryCode": "AUS",
-            "ValidityStartUtc": null,
-            "ValidityEndUtc": null
-        },
         {
             "Code": "AT-2020",
-            "CountryCode": "AUT",
+            "CountryCode": "AT",
+            "TaxationCodes": [
+                "AT-2020",
+                "AT-2020-Extra"
+            ],
             "ValidityStartUtc": "2020-06-30T22:00:00Z",
             "ValidityEndUtc": null
         },
         {
             "Code": "AT-2016",
-            "CountryCode": "AUT",
+            "CountryCode": "AT",
+            "TaxationCodes": [
+                "AT-2016"
+            ]
             "ValidityStartUtc": "2016-05-01T00:00:00Z",
             "ValidityEndUtc": "2020-06-30T22:00:00Z"
         },
         {
             "Code": "AT",
-            "CountryCode": "AUT",
+            "CountryCode": "AT",
+            "TaxationCodes": [
+                "AT"
+            ],
             "ValidityStartUtc": null,
             "ValidityEndUtc": "2016-05-01T00:00:00Z"
         }
-    ],
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `TaxEnvironments` | array of [Tax environment](#tax-environment) | required | The supported tax environments. |
+
+#### Tax environment
+
+Tax environment represents set of [Taxation](#taxation)s together with optional validity interval.
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `Code` | string | required | Code of the tax environment. |
+| `CountryCode` | string | required | ISO 3166-1 alpha-3 code of associated country, e.g. `USA` or `GBR`. |
+| `TaxationCodes` | array of [Taxation](#taxation) codes | required | Codes of the taxations that are used by this environment. |
```

Comment:
I don't think this is how we document foreign keys. It should rather be something like "array of string" And in the description. "Codes of [Taxation](..)s that are used by the environment."